app/uk/gov/hmrc/helptosavefrontend/connectors/NSIConnector.scala

```diff
@@ -63,7 +63,7 @@ class NSIConnectorImpl extends NSIConnector {
       .map { response ⇒
         response.status match {
           case Status.CREATED ⇒
-            Logger.info(s"Successfully created a NSI account for ${userInfo.nino}")
+            Logger.info(s"Received 201 from NSI for create account: ${userInfo.nino}")
```

Comment:
Please change this to say "Received 201 from NSI, successfully created account for ${userInfo.nino}"